front-ps/src/Guide/Routes.purs

```diff
@@ -31,7 +31,7 @@ match url = fromMaybe (NotFound url) $ router url $
   CategoryOverview <<< CategoryName <$> (lit categoryLit *> str) <* end
   <|>
   CategoryDetail <<< CategoryName <$> (lit categoryLit *> str)
-                                  <*> ((\s -> Uid {uidToText : s}) <$> str) <* end
+                                  <*> ((\s -> CUid {uidToText : s}) <$> str) <* end
```

Comment:
Would be nice just to use `CUid s` instead of `CUid {uidToText: s}` (see my other comments)